build-tools/tasks/themeable-source.js

```diff
@@ -2,14 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 const { parallel } = require('gulp');
 const path = require('path');
-const { promises: fsp } = require('fs');
+const { promises: fsp, cpSync } = require('fs');
+const fse = require('fs-extra');
```

Comment:
I do not see this dependency in our `package.json`

Reply:
Hmm, the dependency was available to me. It must be coming in as a transitive. Will add to package json to make it explicit.